build.gradle

```diff
@@ -52,43 +52,50 @@ subprojects {
     // For Google libraries, check <http-client-bom.version>, <google.auth.version>, <guava.version>,
     // ... in https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/pom.xml
     // for best compatibility.
-    GOOGLE_HTTP_CLIENT: "com.google.http-client:google-http-client:1.34.0",
-    GOOGLE_HTTP_CLIENT_APACHE_V2: "com.google.http-client:google-http-client-apache-v2:1.34.0",
-    GOOGLE_AUTH_LIBRARY_OAUTH2_HTTP: "com.google.auth:google-auth-library-oauth2-http:0.18.0",
-    GUAVA: "com.google.guava:guava:28.1-jre",
-    JSR305: "com.google.code.findbugs:jsr305:3.0.2", // transitively pulled in by GUAVA
+    GOOGLE_HTTP_CLIENT: 'com.google.http-client:google-http-client:1.34.0',
+    GOOGLE_HTTP_CLIENT_APACHE_V2: 'com.google.http-client:google-http-client-apache-v2:1.34.0',
+    GOOGLE_AUTH_LIBRARY_OAUTH2_HTTP: 'com.google.auth:google-auth-library-oauth2-http:0.18.0',
+    GUAVA: 'com.google.guava:guava:28.1-jre',
+    JSR305: 'com.google.code.findbugs:jsr305:3.0.2', // transitively pulled in by GUAVA
 
     // for Build Plan and Jib Plugins Extension API
-    BUILD_PLAN: "com.google.cloud.tools:jib-build-plan:0.3.1",
-    EXTENSION_COMMON: "com.google.cloud.tools:jib-plugins-extension-common:0.2.0",
-    GRADLE_EXTENSION: "com.google.cloud.tools:jib-gradle-plugin-extension-api:0.3.0",
-    MAVEN_EXTENSION: "com.google.cloud.tools:jib-maven-plugin-extension-api:0.3.0",
+    BUILD_PLAN: 'com.google.cloud.tools:jib-build-plan:0.3.1',
+    EXTENSION_COMMON: 'com.google.cloud.tools:jib-plugins-extension-common:0.2.0',
+    GRADLE_EXTENSION: 'com.google.cloud.tools:jib-gradle-plugin-extension-api:0.3.0',
+    MAVEN_EXTENSION: 'com.google.cloud.tools:jib-maven-plugin-extension-api:0.3.0',
+
+    COMMONS_COMPRESS: 'org.apache.commons:commons-compress:1.19',
+    JACKSON_DATABIND: 'com.fasterxml.jackson.core:jackson-databind:2.11.0',
+    JACKSON_DATAFORMAT_YAML: 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.11.0',
+    ASM: 'org.ow2.asm:asm:7.3.1',
+    PICOCLI: 'info.picocli:picocli:4.2.0',
+
+    MAVEN_API: 'org.apache.maven:maven-plugin-api:3.5.2',
+    MAVEN_CORE: 'org.apache.maven:maven-core:3.5.2',
+    MAVEN_COMPAT: 'org.apache.maven:maven-compat:3.5.2',
+    MAVEN_ANNOTATIONS_LATEST: 'org.apache.maven.plugin-tools:maven-plugin-annotations:3.6.0',
+    MAVEN_ANNOTATIONS_OLD: 'org.apache.maven.plugin-tools:maven-plugin-annotations:3.5',
+    MAVEN_TESTING_HARNESS: 'org.apache.maven.plugin-testing:maven-plugin-testing-harness:3.3.0',
```

Comment:
can we move 78-79 to the test commented block below?

Reply:
Done.